scripts/mysql_install_db.sh

```diff
@@ -46,6 +46,12 @@ extra_file=""
 dirname0=`dirname $0 2>/dev/null`
 dirname0=`dirname $dirname0 2>/dev/null`
 
+case "$0" in
+  *mysql*)
+    echo "Notice: $0 is deprecated, be MariaDB proud and use mariadb-install-db" 1>&2
```

Comment:
I'm skeptical that the framing of "be MariaDB proud" is going to be useful/clarifying for most users:

1. Most don't know anything about the history of MySQL vs. MariaDB
2. "Be proud" is less directly relevant than "be clear about which database engine you're using"

I would suggest something more like:

> ```
> echo "Notice: the alias $0 is deprecated, in order to prevent confusion over the source of the engine and related scripts and utilities." >&2
> echo "        Use the name $MARIADB_VERSION instead." >&2
> ```

(Perhaps this is a commercial/marketing/branding decision as much as a technical one… maybe @LinuxJedi can weigh in?)

Reply:
Hey @dlenski,

Essentially we are seeing a trend of people who think they are using MySQL but are actually using MariaDB. We want to bring awareness to users that they are using MariaDB and that they should be proud of that. MariaDB is not really a drop in replacement any more and hasn't been for a long time.

Reply:
Understood. (And not surprising given the tools’ naming!)

I'm just suggesting a substantial fraction of users won't have any idea about what "be MariaDB proud" means, nor will it help them understand the difference between MySQL and MariaDB.

Reply:
> Understood. (And not surprising given the tools’ naming!)
> 
> I'm just suggesting a substantial fraction of users won't have any idea about what "be MariaDB proud" means, nor will it help them understand the difference between MySQL and MariaDB.

A good point. We will be making this a wider public effort soon to make both these things more clear.